corehq/apps/hqmedia/views.py

```diff
@@ -136,6 +136,17 @@ def upload_controllers(self):
                                                                        args=[self.domain, self.app_id]))]
 
 
+@method_decorator(require_can_edit_apps, name='dispatch')
+@method_decorator(toggles.BULK_UPDATE_MULTIMEDIA_PATHS.required_decorator(), name='dispatch')
```

Comment:
`require_can_edit_apps` will be checked first and then the toggle right?
I would say, you should check for toggle first since that would be true for much smaller set of users/domain.
Considering the link is already not visible for such domains so this might not be that relevant but just a suggestion.

Reply:
I hadn't thought of that - updated.